ocean_lib/assets/test/test_utils.py

```diff
@@ -49,25 +53,42 @@ def test_utilitary_functions_for_trusted_algorithms(publisher_ocean_instance):
         publisher_ocean_instance.config.metadata_cache_uri,
     )
 
-    assert new_publisher_trusted_algorithms is not None
-    assert len(new_publisher_trusted_algorithms) > len(publisher_trusted_algorithms)
+    assert (
+        new_publisher_trusted_algorithms is not None
+    ), "Added a new trusted algorithm failed. The list is empty."
+    assert len(new_publisher_trusted_algorithms) > len(publisher_trusted_algorithms), (
+        "Added a new trusted algorithm "
+        "failed. The length of the new "
+        "trusted algorithms list should "
+        "be bigger than the previous "
+        "one. "
+    )
 
     # add an existing algorithm to publisher_trusted_algorithms list
     new_publisher_trusted_algorithms = add_publisher_trusted_algorithm(
         ddo.did, algorithm_ddo.did, publisher_ocean_instance.config.metadata_cache_uri
     )
     assert new_publisher_trusted_algorithms is not None
     for _, trusted_algorithm in enumerate(publisher_trusted_algorithms):
```

Comment:
please do not use enumerate if you are not using the key, use a simple `for` statement instead.